src/Views/TryInstallView.vala

```diff
@@ -55,14 +55,32 @@ public class Installer.TryInstallView : AbstractInstallerView {
         title_grid.row_spacing = 6;
         title_grid.halign = Gtk.Align.CENTER;
         title_grid.valign = Gtk.Align.CENTER;
-        title_grid.attach (type_image, 0, 0, 1, 1);
-        title_grid.attach (type_label, 0, 1, 1, 1);
-        title_grid.attach (type_desc_label, 0, 2, 1, 1);
+
+        title_grid.attach (type_image,      0, 0);
+        title_grid.attach (type_label,      0, 1);
+        title_grid.attach (type_desc_label, 0, 2);
+
+        // TODO: Once we support more options, give an example here
+        // ("More options, such as…") if there's space…
+        var decrypt_description = new Gtk.Label (_("More options may be available after unlocking encrypted drives"));
+
+        var decrypt_button = new Gtk.Button.with_label (_("Unlock Encrypted Drives…"));
+
+        // TODO: Only show decrypt infobar when we detect encrypted drives
```

Comment:
In https://github.com/pop-os/installer/pull/126, this will soon be possible through `InstallOptions.get_default ().contains_luks ()`. Then for getting a list of devices to decrypt, I'll add a `borrow_disks ()` method which you could use to check partitions with a file system of `LUKS`, and for attempting the decryption.